src/backend/db/videos.js

```diff
@@ -88,13 +86,13 @@ export const videos = [
         creator: "Freethink",
         icon: "freethink.jpeg",
         thumbnail: "KfB2sx9uCkI-MQ.jpg",
-        url: "https://www.youtube.com/watch?v=KfB2sx9uCkI",
+        embedId: "KfB2sx9uCkI",
         views: "50k",
         uploadedOn: "1 year ago",
         category: "Documentary",
     },
     {
-        _id: uuid(),
+        _id: 'xpmy12-k8fqxp-lwaph1qj',
```

Comment:
is uuid is not working here

Reply:
uuid is not required for my functionality